client/src/App.js

```diff
@@ -70,23 +75,57 @@ function App() {
     setShowAddModal(false)
   }
 
-  const onSuccessAddSubscriber = () => {
+  const onSuccessAddSubscriber = (success) => {
     setShowAddModal(false)
+    refreshSubscribers()
+    setAlertMessage(success)
+    setAlertVariant('success')
+    window.setTimeout(() => {
+      setAlertMessage('')
+      setAlertVariant('')
+    }, 5000)
+  }
+
+  const onFailureAddSubscriber = (error) => {
+    setAlertMessage(error)
+    setAlertVariant('danger')
+    window.setTimeout(() => {
+      setAlertMessage('')
+      setAlertVariant('')
+    }, 5000)
   }
 
   const onUpdateStatusSelectected = (subscriberId, status) => {
     setFocusedSubscriberId(subscriberId)
-    setFocusedSubscriberStatus(status)
+    setFocusedSubscriberStatus(status.toString())
   }
 
   const onCloseUpdateStatusSubscriberModal = () => {
     setFocusedSubscriberId('')
     setFocusedSubscriberStatus('')
   }
 
-  const onSuccessUpdateStatusSubscriber = () => {
+  const onSuccessUpdateStatusSubscriber = (success) => {
```

Comment:
This function now refreshes the subscriber index table and shows a success alert when a subscriber's status has been successfully updated